use-cases/client-server-msgs-perf/svmsg_file_server.c

```diff
@@ -412,25 +408,10 @@ main(int argc, char *argv[])
             // One client has informed that it has exited
             NumActiveClients--;
 
-            // NOTE: If server is started using one of the --clock arguments,
-            // the metric's name logged in L3's log-line, above, will be
-            // misleading. It needs to be hard-coded to a literal string at
-            // compile-time. The metric's name given below is the right one
-            // corresponding to the active clock.
-            size_t throughput = 0;
```

Comment:
This comment is no longer valid or relevant. It's obsoleted by the rework done for perf metrics tracking under SHA 2d6b9d6